### PR TITLE
Add dropdown for Template Item

### PR DESCRIPTION
**Pull request by not7cd**
image

### CODE REVIEW

File: froncik/.env.local [hunk 1]
```diff
@@ -1 +1 @@
-# NEXT_PUBLIC_API_PATH=http://127.0.0.1:1234/majooo #default /api
+NEXT_PUBLIC_API_PATH=http://127.0.0.1:8080/api #default /api
```
Comment (oneacik) [inline froncik/.env.local]: As the DEV.md states for running:
Wed 30 Nov 15:57:18 CET 2022

Changing this line will probably break it for the production application.
I may fix it today to modify this variable for production.

For now please remove.

File: froncik/components/TemplateCard.tsx [hunk 2]
```diff
@@ -23,7 +23,21 @@ export default function TemplateCard({ template }: Props) {
...
     >
-      <Card.Header><a href={`./api/templates/${template.id}/download`}>{template.name}</a></Card.Header>
+      <Card.Header>
+        <Stack direction="horizontal" gap={3}>
+          <h4>{template.name}</h4>
+          <Dropdown className="ms-auto">
+            <Dropdown.Toggle variant="secondary" id="dropdown-basic">
+              
+            </Dropdown.Toggle>
+            <Dropdown.Menu>
```
Comment (oneacik) [inline froncik/components/TemplateCard.tsx:33]: nice

File: froncik/components/TemplateCard.tsx [hunk 3]
```diff
@@ -40,15 +54,6 @@ export default function TemplateCard({ template }: Props) {
...
-              style={buttonStyle}
-              onClick={() => API.templates.delete(template.id)}
-            >
-              Usuń
-            </Button>
-            <Link href={routes.edit(template.id)} passHref legacyBehavior>
-              <Button style={buttonStyle}>Edytuj</Button>
-            </Link>
             <Link href={routes.print(template.id)} passHref legacyBehavior>
               <Button style={buttonStyle}>Drukuj</Button>
```
Comment (oneacik) [inline froncik/components/TemplateCard.tsx:58]: Optional: Change it to "Przygotuj do Druku"

**Review by oneacik — APPROVED** [review]
You get approve, just please remove .env.local change

File: froncik/components/TemplateCard.tsx (hunk 3 above)
Comment (oneacik) [reply froncik/components/TemplateCard.tsx:58]: Or even if you are cool enough, the whole image may be a link

Comment (not7cd) [reply froncik/components/TemplateCard.tsx:58]: ACK

File: froncik/components/TemplateCard.tsx (hunk 2 above)
Comment (not7cd) [reply froncik/components/TemplateCard.tsx:33]: ACK

File: froncik/.env.local (hunk 1 above)
Comment (not7cd) [reply froncik/.env.local]: DONE

**Pull request merged**